x-pack/plugins/transform/public/app/hooks/use_refresh_transform_list.ts

```diff
@@ -15,6 +15,7 @@ export const useRefreshTransformList = () => {
   return () => {
     queryClient.invalidateQueries([TRANSFORM_REACT_QUERY_KEYS.GET_TRANSFORM_NODES]);
     queryClient.invalidateQueries([TRANSFORM_REACT_QUERY_KEYS.GET_TRANSFORMS]);
+    queryClient.invalidateQueries([TRANSFORM_REACT_QUERY_KEYS.GET_TRANSFORMS_STATS]);
```

Comment:
This now needs to be `GET_TRANSFORM_STATS` otherwise the individual requests won't get invalidated.

Reply:
Updated here [`e45483c` (#166320)](https://github.com/elastic/kibana/pull/166320/commits/e45483cf9b0cb3deddb9c14ef4816a52c628244e)

Reply:
Reverted this update now that we are refeteching all stats together again